apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -329,6 +331,8 @@ plugins(_) ->
             {"plugins <command> [Name-Vsn]", "e.g. 'start emqx_plugin_template-5.0-rc.1'"},
             {"plugins list", "List all installed plugins"},
             {"plugins describe  Name-Vsn", "Describe an installed plugins"},
+            {"plugins allow     Name-Vsn",
```

Comment:
thinking about it again, should there perhaps be a `disallow` command ?

Reply:
I imagined it should

Reply:
:checkered_flag: